reference/docs-conceptual/learn/remoting/SSH-Remoting-in-PowerShell-Core.md

```diff
@@ -71,14 +71,14 @@ an SSH subsystem to host a PowerShell process on the remote computer. And, you m
 
    Make sure password authentication is enabled:
 
-   ```
+   ```Output
```

Comment:
This is a configuration setting in a file

Reply:
dixed